api/src/main/java/datawave/security/authorization/DatawaveUser.java

```diff
@@ -37,17 +39,25 @@
     
```

Comment:
Backward compatible constructor that excludes the email parameter.  If the other Datawave repos that use this form get refactored, then this can be removed.